drivers/hmis_data_quality_tool/app/models/hmis_data_quality_tool/enrollment.rb

```diff
@@ -209,8 +228,9 @@ def report_item_fields_from_enrollment(report_items:, enrollment:, report:) # ru
       report_item.data_source_id = enrollment.data_source_id
       report_item.project_id = enrollment.project.id
       report_item.project_name = enrollment.project.name(report.user)
-      project_type =  enrollment.project.project_type_to_use
+      project_type = enrollment.project.project_type
       report_item.project_type = project_type
+      report_item.funders = enrollment.project.funders.pluck(:Funder).map(&:to_i)
```

Comment:
Hmm, I'm not sure how rails handles a pluck like this.  I know you can pluck from an array (and a scope) so it might do the right thing and just grab it out of the array, but it might be safer (from an N+1 perspective, since you've already preloaded) to just:
```suggestion
      report_item.funders = enrollment.project.funders.map(&:Funder).reject(&:blank).map(&:to_i)
```
I'm also not sure you want to call `to_i` on it without first rejecting the empty values.  If you don't you'll probably get some unintentional `0`s, since `nil.to_i` and `''.to_i` result in `0`.